pills/03-enter-environment.xml

```diff
@@ -41,20 +41,19 @@
 
     <para>
       Finally something practical! Installation into the Nix environment is an
-      interesting process. Let's install <literal>nix-repl</literal>, a simple command line tool
-      for playing with the Nix language. Yes, Nix is a <link xlink:href="http://nixos.org/nix/manual/#idm47361539226272">
-        pure, lazy, functional language</link>, not only a set of tools to
-      manage derivations.
+      interesting process. Let's install <link xlink:href="https://github.com/bennofs/nix-index#nix-index-">
+      <literal>nix-index</literal></link>, a simple command line tool
+      to quickly locate files in any store path.
     </para>
 
     <para>
       Back to the installation:
     </para>
 
-    <screen><xi:include href="./03/install-repl.txt" parse="text" /></screen>
+    <screen><xi:include href="./03/install-nix-index.txt" parse="text" /></screen>
 
     <para>
-      Now you can run <literal>nix-repl</literal>. Things to notice:
+      Now you can run <literal>nix-index</literal>. Things to notice:
```

Comment:
nix-index takes a long time to run and may confuse people even more, why not install something simple, like the package `hello`?